src/cba-list-new/cba-list-new.js

```diff
@@ -118,10 +128,11 @@ class List extends HTMLElement
     this.tooltipLinkText = this.getAttribute("tooltip-link-text");
     this.tooltipLinkTextDefault = "Learn more";
     this.connected = true;
+    this.group = this.shadowRoot.querySelector("#group");
 
     this.container.addEventListener("click", ({target}) =>
     {
-      if (target.tagName === "BUTTON")
+      if (target.classList.contains("collapsed") || target.classList.contains("expanded"))
```

Comment:
We could reffer to the button DOM via styles. - example -  if btn arrow is rotated (transform) we can apply specific changes.

Reply:
The question here is about action handling, we currently are using listener on `this.container` for expanding and collapsing and using `@click` lit-html property for other actions like `+` buttons, I am suggesting to figure out why one is beneficiar than the other.

So we might either want to move this to using `@click` for those items, or move those that are using `@click` to here, for that we can have for example a data attribute which will suggest one or another action (i.e..: `data-action="collapse"`, `data-action="addItem"`).

The reason why we might want to be in favor of using 1 action listener, is that it may contribute to the performance when there will be a lot of actions (ex.: 1000+), make sense to investigate if performance is the issue here, but otherwise having top level listener is sounds like most safe option.

Reply:
For the reference: https://lit.dev/docs/components/events/#event-delegation